src/go/configgenerator/filterconfig/filter_generator.go

```diff
@@ -266,6 +275,19 @@ func updateProtoDescriptor(service *confpb.Service, apiNames []string, descripto
 	return newData, nil
 }
 
+func preserveDefaultHttpBinding(httpRule *ahpb.HttpRule, defaultPath string) {
+	defaultBindingExisted := false
+	defaultBinding := &ahpb.HttpRule{Pattern: &ahpb.HttpRule_Post{defaultPath}, Body: "*"}
+	for _, addtionalBinding := range httpRule.AdditionalBindings {
```

Comment:
also compare with the primary binding other than additional bindings also

Reply:
Fixed.